ng/src/web/components/table/head.js

```diff
@@ -38,20 +40,48 @@ const TableHead = ({
     className,
     colSpan,
     rowSpan,
-    sortby,
+    sortDir,
+    filterSortBy,
+    sortBy,
     width,
     onSortChange,
     ...other
   }) => {
+
+  const sortSymbol = sortDirection => {
```

Comment:
Any reason why you did put this into a function? It can just be a variable because the param sortDirection is static and provided via props already.